lib/templates/middleAssignment/pages/homescreen.dart

```diff
@@ -0,0 +1,135 @@
+import 'dart:async';
+import 'package:flutter/material.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+
+class MainScreen extends StatelessWidget {
+  static const String firstKey = "first";
+  static const String secondKey = "second";
+  static const String thirdKey = "third";
+
+  String sesF = '';
+  String sesS = '';
+  String sesT = '';
+
+  MainScreen({super.key});
+
+  Future<String> getFirst() async {
+    var prefs = await SharedPreferences.getInstance();
+    return prefs.getString(firstKey) ?? "No prev session";
+  }
```

Comment:
Not changing

![Screenshot 2024-06-24 at 16 07 51](https://github.com/timur-harin/Sum24FlutterCourse/assets/64702874/6c7ee1b6-e14b-4a66-9aa2-be6a78ce7ba8)
![Screenshot 2024-06-24 at 16 07 54](https://github.com/timur-harin/Sum24FlutterCourse/assets/64702874/e9dfdff0-c3ae-4a77-8c6b-5f6a3eb0db2e)